### ISSUE TITLE
Loss = inf after few epochs

### ISSUE THREAD

**Issue by sruteesh** [mozilla/DeepSpeech#845]
Hello,
We are training speech recogntion using our own corpus using modified version of `run-librivox.sh` (varied the batch_size)  with warp_ctc loss function. 
I encountered `loss=inf` at 3rd epoch (1 and 2 ran without any issue). PFA the log file.
[infloss.txt](https://github.com/mozilla/DeepSpeech/files/1318520/infloss.txt)

I have also tried to decrease the learning rate (`lr= 0.000001`) but the error still continues. Can u please explain what exactly causes the loss to become inf. Neither of my labels nor wav files are empty.
